cmd/kueueviz/OWNERS

```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- mbobrovskyi
+- mimowo
+- tenzen-y
+
+reviewers:
+- akram
```

Comment:
Please update the PR to only nominate yourself. Also, be aware that the ownership is inherited. So, no need to repeat other approaves or reviewers

Reply:
@akram could you address this comment.

Reply:
sure